examples/platform/silabs/uart.cpp

```diff
@@ -138,7 +138,13 @@ constexpr osThreadAttr_t kUartTaskAttr = { .name       = "UART",
                                            .cb_size    = osThreadCbSize,
                                            .stack_mem  = uartStack,
                                            .stack_size = kUartTaskSize,
-                                           .priority   = osPriorityRealtime };
+#if SLI_SI91X_MCU_INTERFACE
+                                           // Reducing the priority of the UART task to avoid priority inversion
```

Comment:
priority inversion over what task? This task only depends on what is placed on its queue and then writes it to uart.

Reply:
noticed that the PASE session itself is not getting started if the logs are more, if LWIP logs are also enabled the issue is seen in CASE session. Therefore the UART logging priority is reduced from the BLE and LWIP task so that logging never blocks the important task